add update and details features

lluminita opened the pull request:
solves issues: #45 #44 #43 

Former closed PR: https://github.com/lluminita/codeeve/pull/46

lluminita committed:
add update and details features

mei-li commented on projects/templates/projects/project_details.html:
```diff
@@ -0,0 +1,51 @@
+                    <h2 class="section-heading">{{ project.title }}</h2>
+                    <p>{{ project.created_at|date:"D d M Y"  }}</p>
+                    <div
+                      {% if project.difficulty == "beginner" %}
```
I think you can do
`<div class=glossy-{{ project.difficulty }}>`

mei-li commented on projects/templates/projects/update_project.html:
```diff
@@ -0,0 +1,30 @@
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <form action="{% url 'project_details' pk=project.pk %}" method="post" enctype="multipart/form-data">
```
btw it says here https://docs.djangoproject.com/en/1.9/topics/http/file-uploads/ to check with security on uploading, we should do it when deploying. Maybe heroku has it already though

lluminita committed:
fix comments and remove template_name from view

lluminita commented:
I fixed the _difficulty_ comment and I remove the create and update templates and replaced with project_form, this way there is no need to use the attribute template_name in the views.

mei-li commented:
I made this issue for the owner https://github.com/lluminita/codeeve/issues/52

mei-li commented:
:+1:
